framework/src/media/audio/audio_manager.c

```diff
@@ -1478,29 +1793,72 @@ void print_audio_card_info(audio_io_direction_t direct)
 #if defined(CONFIG_DEBUG) && defined(CONFIG_DEBUG_MEDIA)
 	audio_card_info_t *card;
 	audio_card_status_t status;
-	int i, j, max_card, actual_card;
-	char path[AUDIO_PCM_FULL_PATH_LENGTH];
+	int i, j, max_card, actual_card_id;
+	char path[AUDIO_DEVICE_FULL_PATH_LENGTH];
 
 	if (direct == INPUT) {
 		max_card = CONFIG_AUDIO_MAX_OUTPUT_CARD_NUM;
+		actual_card_id = g_actual_audio_in_card_id;
 	} else {
 		max_card = CONFIG_AUDIO_MAX_INPUT_CARD_NUM;
+		actual_card_id = g_actual_audio_out_card_id;
 	}
 
 	for (j = 0; j < CONFIG_AUDIO_MAX_DEVICE_NUM; j++) {
 		for (i = 0; i < max_card; i++) {
 			if (direct == INPUT) {
 				card = &g_audio_in_cards[i];
-				actual_card = g_actual_audio_in_card_id;
 			} else {
 				card = &g_audio_out_cards[i];
-				actual_card = g_actual_audio_out_card_id;
 			}
 			status = card->config[j].status;
 			if (status != AUDIO_CARD_NONE) {
 				get_card_path(path, i, j, direct);
-				dbg_noarg("\nPath %s\nCard Number : %d\nDevice Number : %d\nStatus : %d\nPolicy : %d\n", path, i, j, status, card->policy);
-				dbg_noarg("Volume : %d\nMax Volume : %d\nSupports Process Type : %d\n", card->config[j].volume, card->config[j].max_volume, card->config[j].device_process_type);
+
+				dbg_noarg("\nDevice Path : %s\n", path);
```

Comment:
Is it a config newly added?
Or this needs to be changed into medvdbg...?

Reply:
dbg_noarg do not print out function name, that;'s why I added this as a print out info.
Also developer can decide use this or not by calling this method, so verbose is not good option.
Please see 1794 line, I already added condition.